### PR TITLE
update the install docs with community operator

### PR DESCRIPTION
**Pull request by savitharaghunathan**

### CODE REVIEW

File: content/Konveyor/installation.md [hunk 1]
```diff
@@ -22,43 +22,32 @@ Follow the steps below to provision minikube for single users deploying Konveyor
...
-If you need to deploy a beta release (or special branch) please replace the *main* branch in URL with the desired beta branch (i.e v2.0.0-beta.0):
+### Installing the latest
 
-`$ kubectl apply -f https://raw.githubusercontent.com/konveyor/tackle2-operator/<release>-beta.0/tackle-k8s.yaml`
+If you need to deploy a latest build off main please use the below url
 
-**Note:** Upgrades between beta releases are **not guaranteed** , once installed, we strongly suggest to edit your subscription and switch to Manual upgrade mode for beta releases: `$ kubectl edit subscription` -> installPlanApproval: Manual
+```
+[user@user ~]$ kubectl create -f https://operatorhub.io/install/alpha/konveyor-operator.yaml
+```
```
Comment (jwmatthews) [inline content/Konveyor/installation.md:49]: alpha isn't really 'latest' tracking main, alpha are more pre-release builds.

To install ':latest' which tracks main we can use:
https://raw.githubusercontent.com/konveyor/tackle2-operator/main/tackle-k8s.yaml


The diff is that we need to create a custom CatalogSource using
quay.io/konveyor/tackle2-operator-index:latest

```
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: konveyor
  namespace: konveyor-tackle
spec:
  displayName: Konveyor Operator
  publisher: Konveyor
  sourceType: grpc
  image: quay.io/konveyor/tackle2-operator-index:latest
```

and then a Subscription using the custom catalog source

```
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: konveyor-operator
  namespace: konveyor-tackle
spec:
  channel: development
  installPlanApproval: Automatic
  name: konveyor-operator
  source: konveyor
  sourceNamespace: konveyor-tackle


```

**Review by jwmatthews — CHANGES_REQUESTED** [review]
I think we should update the alpha section.

Right now we say alpha is installing latest tracking main which isn't really true.
alpha is for pre-release builds

Maybe to avoid confusion we should call out all the paths we have?
konveyor-0.1
alpha
:latest

This will mean that :latest is a custom catalog source, and konveyor-0.1 and alpha are coming from community-operators.

File: content/Konveyor/installation.md [hunk 2]
```diff
@@ -22,43 +22,76 @@ Follow the steps below to provision minikube for single users deploying Konveyor
...
-|keycloak postgresql|1Gi|RWO|Keycloak backend DB|
-|pathfinder postgresql|1Gi|RWO|Pathfinder backend DB|
-|maven|100Gi|RWX|maven m2 repository|
-
-Follow the steps below to install the Konveyor Operator in the konveyor-tackle namespace (default) on any Kubernetes distribution, including minikube.
+Follow the steps below to install the Konveyor Operator in the `my-konveyor-operator` namespace (default) on any Kubernetes distribution, including minikube.
 
 **Procedure**
-1. Install the Konveyor Operator.
+{{< tabs >}}
```
Comment (jwmatthews) [inline content/Konveyor/installation.md:34]: Use of tabs is AWESOME, great improvement in making it clean!

File: content/Konveyor/installation.md [hunk 3]
```diff
@@ -22,43 +22,76 @@ Follow the steps below to provision minikube for single users deploying Konveyor
...
+spec:
+  channel: development
+  installPlanApproval: Automatic
+  name: konveyor-operator
+  source: konveyor
+  sourceNamespace: konveyor-tackle
+```
+* Create the Tackle instance using the url below,
+```
+kubectl create -f https://raw.githubusercontent.com/konveyor/tackle2-operator/main/tackle-k8s.yaml
```
Comment (jwmatthews) [inline content/Konveyor/installation.md]: Note we just want the Tackle CR created here.

The referenced file has different content we don't want for this step:

https://raw.githubusercontent.com/konveyor/tackle2-operator/main/tackle-k8s.yaml

```
---
apiVersion: v1
kind: Namespace
metadata:
  name: konveyor-tackle
---
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: konveyor
  namespace: konveyor-tackle
spec:
  displayName: Konveyor Operator
  publisher: Konveyor
  sourceType: grpc
  image: quay.io/konveyor/tackle2-operator-index:latest
---
apiVersion: operators.coreos.com/v1
kind: OperatorGroup
metadata:
  name: konveyor
  namespace: konveyor-tackle
spec:
  targetNamespaces:
    - konveyor-tackle
---
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: konveyor-operator
  namespace: konveyor-tackle
spec:
  channel: development
  installPlanApproval: Automatic
  name: konveyor-operator
  source: konveyor
  sourceNamespace: konveyor-tackle
```

**Review by jwmatthews — CHANGES_REQUESTED** [review]
Need to update Tackle CR for the case of using latest

File: content/Konveyor/installation.md [hunk 4]
```diff
@@ -22,43 +22,76 @@ Follow the steps below to provision minikube for single users deploying Konveyor
...
+  name: konveyor-operator
+  namespace: konveyor-tackle
+spec:
+  channel: development
+  installPlanApproval: Automatic
+  name: konveyor-operator
+  source: konveyor
+  sourceNamespace: konveyor-tackle
+```
+* Create the Tackle instance using the url below,
```
Comment (jwmatthews) [inline content/Konveyor/installation.md]: I think you will want to delete this step as you have it covered later

**Review by jwmatthews — APPROVED** [review]
LGTM

**Comment by savitharaghunathan** [comment]
Thank you @jwmatthews

**Pull request merged**